content/gotraining.slide

```diff
@@ -488,15 +488,21 @@ As part of all labs
 
 * Functions
 
-- Functions
-- Multiple Return Values
-- Variadic Functions
-- Closures
+- [[https://goplay.space/#M_EBwjCDT77][Functions]]: parameters and return values
+- [[https://goplay.space/#Kc-7LOUq_Ls][Variadic Functions]]
+- [[https://goplay.space/#olrpJoSte3n][Closures]]
 
 
-* Lab 4
+* Lab 4 - numeronym
 
-TODO
+- Create an executable go program in directory `3_letters`
+- Write a function that returns a slice of numeronyms for its input strings
+
+	func numeronyms(vals ...string) []string
+
+- Call `fmt.Prinln(numeronyms("accessibility",` `"Kubernetes",` `"abc"))` in `main` to print
+
+	[a11y K8s abc]
```

Comment:
a1c? :)  what's the lowest number you put in - 2 I guess. perhaps the class can discuss this.

Reply:
yeah, good idea to discuss.... i'd say it should shorten the input....

Reply:
definitely should shorted, but a minimum amount? you probably wouldn't want to do d2e for dude, for example. but they can try coding up a solution where a var or constant controls it, for a bit of extra work.